src/mpcCoreKit.ts

```diff
@@ -119,22 +119,33 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     } else log.setLevel("error");
     if (typeof options.manualSync !== "boolean") options.manualSync = false;
     if (!options.web3AuthNetwork) options.web3AuthNetwork = WEB3AUTH_NETWORK.MAINNET;
+    // we accept sessionTime to be 0 which will disable the session manager creation
+    // if sessionTime is not provided, it is defaulted to 86400
     if (!options.sessionTime) options.sessionTime = 86400;
     if (!options.serverTimeOffset) options.serverTimeOffset = 0;
     if (!options.uxMode) options.uxMode = UX_MODE.REDIRECT;
     if (!options.redirectPathName) options.redirectPathName = "redirect";
     if (!options.baseUrl) options.baseUrl = isNodejsOrRN ? "https://localhost" : `${window?.location.origin}/serviceworker`;
     if (!options.disableHashedFactorKey) options.disableHashedFactorKey = false;
     if (!options.hashedFactorNonce) options.hashedFactorNonce = options.web3AuthClientId;
+    if (options.enableSessionManager === undefined) options.enableSessionManager = true;
```

Comment:
i think changing name to `disableSessionManager` makes more sense, since its enabled by default.